addons/mail/static/tests/owl/form_renderer_chatter_tests.js

```diff
@@ -0,0 +1,288 @@
+                    message_follower_ids: {
+                        string: "Followers",
+                        type: "one2many",
+                        relation: 'mail.followers',
+                        relation_field: "res_id",
+                    },
+                    message_ids: {
+                        string: "messages",
+                        type: "one2many",
+                        relation: 'mail.message',
+                        relation_field: "res_id",
+                    },
+                    activity_ids: {
+                        string: 'Activities',
+                        type: 'one2many',
+                        relation: 'mail.activity',
+                        relation_field: 'res_id',
+                    },
+                    activity_exception_decoration: {
+                        string: 'Decoration',
+                        type: 'selection',
+                        selection: [['warning', 'Alert'], ['danger', 'Error']],
+                    },
+                    activity_exception_icon: {
+                        string: 'icon',
+                        type: 'char',
+                    },
+                    activity_state: {
+                        string: 'State',
+                        type: 'selection',
+                        selection: [['overdue', 'Overdue'], ['today', 'Today'], ['planned', 'Planned']],
+                    },
+                    message_attachment_count: {
+                        string: 'Attachment count',
+                        type: 'integer',
+                    },
+                },
+                records: [{
+                    id: 1,
+                    message_attachment_count: 0,
+                    display_name: "first partner",
+                    foo: "HELLO",
+                    message_follower_ids: [],
+                    message_ids: [],
+                    activity_ids: [],
+                }, {
+                    id: 2,
+                    message_attachment_count: 0,
+                    display_name: "second partner",
+                    foo: "HELLO BITCH",
+                    message_follower_ids: [],
+                    message_ids: [],
+                    activity_ids: [],
+                }]
+            }
+        };
+        this.createView = async (...args) => {
+            const { widget } = await start(...args);
+            this.view = widget;
+            await afterNextRender();
+        };
+    },
+    afterEach: function () {
+        _.debounce = this.underscoreDebounce;
+        _.throttle = this.underscoreThrottle;
+        if (this.view) {
+            this.view.destroy();
+        }
+    }
+});
+
+QUnit.test('basic chatter rendering', async function (assert) {
+    assert.expect(1);
+    await this.createView({
+        hasView: true,
+        async mockRPC(route, args) {
+            const _super = this._super.bind(this, route, args); // limitation on class.js with async/await
```

Comment:
Limitation not applied here. Could simply call `this._super(...args);` when necessary.